src/editor.js

```diff
@@ -1257,6 +1296,124 @@ export class OdooEditor extends EventTarget {
         }
     }
 
+    // PASTING / DROPPING
+
+    /**
+     * Prepare clipboard data (text/html) for safe pasting into the editor.
+     *
+     * @private
+     * @param {string} clipboardData
+     * @returns {string}
+     */
+     _prepareClipboardData(clipboardData) {
+        const container = document.createElement('fake-container');
+        container.innerHTML = clipboardData;
```

Comment:
Security is going to scream. Paste html with a script in it and poof, it's executed now.

Reply:
Well sure... but it's adding that innerHTML to a container we don't directly add to the dom, and that we then clean. They had validated that method for we3.